python/paddle/fluid/tests/unittests/test_lookup_table_bf16_op.py

```diff
@@ -25,18 +25,21 @@
 from paddle import enable_static
 
 
-def _lookup(weights, ids, flat_ids):
+def _lookup(weights, ids, flat_ids, op_type="lookup_table"):
```

Comment:
Maybe "op_version" would be better? Now it suggests that this function is for different operator types.